translations/instadefuse.phrases.txt

```diff
@@ -23,4 +23,10 @@
         "#format"    "{1:s},{2:.1f}"
         "en"        "{1} There were {2} seconds left of the bomb. Counter Terrorists win."
     }
+	
+    "SomeoneAliveHaveKit"
+    {
+        "#format"    "{1:s}"
+        "en"        "{1} There's no time to defuse without a Defuse Kit!"
```

Comment:
`"{1} There's no time to defuse without a Defuse Kit!"` to `"{1} You have no time! Your teammate has a Defuse Kit!"`